### PR TITLE
Done changes in data saving and ui

### PR DESCRIPTION
**Pull request by Ritika8081**
- [x] buttons alignment 
- [x] data saving in indexed db 
- [x] grid/list view
- [x] stop data on screen
- [x] file download and delete before start recording again

### CODE REVIEW

**Comment by Ritika8081** [comment]
I have included aman's changes in this pull request #10 .

File: .env [hunk 1]
```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_VERSION=2.0.0
```
Comment (akadeepesh) [inline .env]: Please create .env.local file instead of .env, and make sure to not push it on github.

File: package.json [hunk 2]
```diff
@@ -29,6 +29,7 @@
     "@types/w3c-web-serial": "^1.0.6",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.1",
+    "dotenv": "^16.4.5",
```
Comment (akadeepesh) [inline package.json]: Remove this package, we don't need this.

File: src/components/Contributors.tsx [hunk 3]
```diff
@@ -12,7 +12,12 @@ import { Dialog, DialogContent, DialogTrigger } from "./ui/dialog";
 import { CircleAlert } from "lucide-react";
 import { Button } from "./ui/button";
 import Link from "next/link";
+import dotenv from "dotenv";
```
Comment (akadeepesh) [inline src/components/Contributors.tsx]: Remove this, you can access NEXT_EXAMPLE variable from env like process.env.NEXT_EXAMPLE. variable name can be anything this is just an example.

File: src/components/Contributors.tsx [hunk 4]
```diff
@@ -62,6 +67,7 @@ const Contributors = () => {
         <DialogContent onOpenAutoFocus={(e) => e.preventDefault()}>
           <Card className="border-none -m-5">
             <CardHeader>
+              <p>Chords:{Version}</p>
```
Comment (akadeepesh) [inline src/components/Contributors.tsx:66]: I recommend to put software name in env as well

**Review by akadeepesh — CHANGES_REQUESTED** [review]

**Comment by akadeepesh** [comment]
Mark resolve conversation after doing changes.

File: src/components/Contributors.tsx (hunk 4 above)
Comment (Ritika8081) [reply src/components/Contributors.tsx:66]: I have done changes, please review.

**Review by akadeepesh — APPROVED** [review]

**Comment by akadeepesh** [comment]
I'm merging this to main :+1:

**Pull request merged**